content/docs/upgrading/upgrade.md

```diff
@@ -6,11 +6,8 @@ weight = 30
 
 ## Upgrading from Kubeflow v0.7.0 or later
 
-{{% alert title="Alpha version" color="warning" %}}
-This feature is currently in <b>Alpha</b> with limited support. The Kubeflow
-team is interested in any feedback you may have, in particular with regards to
-usability of the feature.
-{{% /alert %}}
+{{% alpha-status 
+  feedbacklink="https://github.com/kubeflow/mpi-operator/issues" %}}
```

Comment:
Let's point this feedback link at the issue tracker for `kubeflow/kubeflow` too:

```
feedbacklink="https://github.com/kubeflow/kubeflow/issues"
```

Reply:
Thanks @sarahmaddox for review again. I have modified the files as per your review feedback. Please review/approve. Feel free to let me know if further change is needed.